### DISCUSSION TITLE
How do I install OBS Background remover?

### DISCUSSION THREAD

**Discussion by rtperrett** [royshil/obs-backgroundremoval#307] [General]
How do I install OBS Background remover? I have OBS 29.1.1 installed on Ubuntu Linux 22.04.2 LTS. I type flatpak install com.obsproject.Studio.Plugin.BackgroundRemoval in Terminal, unless I am missing a step according to the instructions at https://github.com/royshil/obs-backgroundremoval

**Comment by umireon** [comment]
If you have installed OBS via FlatHub, you need to install this plugin via FlatHub.
If you have installed OBS via PPA (apt install obs), you need to download https://github.com/royshil/obs-backgroundremoval/releases/download/v0.5.19/obs-backgroundremoval-0.5.19-linux-x86_64.deb and install this deb package.

After the installation, you can see this plugin on Effect filters.

**Reply by rtperrett** [reply]
It worked, thank you. I will be able to do videos.
